hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/recordcache/RecordCacheService.java

```diff
@@ -34,7 +34,7 @@ public class RecordCacheService implements Service {
     /** The record cache service name */
     public static final String NAME = "RecordCache";
     /** The name of the queue that stores the transaction records */
-    static final String TXN_RECORD_QUEUE = "TransactionRecordQueue";
+    public static final String TXN_RECORD_QUEUE = "TransactionRecordQueue";
```

Comment:
I'm not sure if this change was needed, did it need to be made public?

Reply:
I probably pushed that with some other changes. Currently, I'm using it on the new test I start writing but it's not pushed yet. I will wait for the story to be finalized so I can finish and push that test.

Reply:
Is it necessary to make it public? I fight to keep things as restricted in scope as possible. It helps with readability and safety. A public field might be used by anywhere, so I have to "find usages" to figure out who depends on it and why. If it is package private, I know at least it is only accessed within the package, so I can immediately mentally eliminate most of the code. I stress out changing something from private to package private for the sake of testing (which I hardly ever will do), I would really try very hard to avoid making something public for the sake of testing.